lib/foreman/silenced_logger.rb

```diff
@@ -3,6 +3,11 @@
 # assets logging feature.
 module Foreman
   class SilencedLogger < SimpleDelegator
+    # Allow webpack to run logger.tagged
+    def tagged(*tags)
+      yield self
+    end
```

Comment:
It seems that our logger is outdated, webpacker is running:
```ruby
  initializer "webpacker.logger" do
    config.after_initialize do
      if ::Rails.logger.respond_to?(:tagged)
        Webpacker.logger = ::Rails.logger
      else
        Webpacker.logger = ActiveSupport::TaggedLogging.new(::Rails.logger)
      end
    end
  end

Webpacker.logger.tagged("Webpacker") { compiler.compile }
```

Reply:
should we send an upstream patch to support this on the logging gem?

Reply:
I am not sure about it, check out this file:
https://github.com/theforeman/foreman/blob/develop/lib/foreman/logging.rb#L53-L56

Why do we need so many loggers? Is it possible that this function tries to achieve the same behavior as `tagged`?